PDCMFinder/cancer-models
# Adding storybook to project

## Issue
Addresses #9  

## Description

Adds Storybook to the NextJS project. This has a temporary fix to an issue generated by the latest release of Storybook: https://github.com/webpack/enhanced-resolve/issues/362
After that Issue is solved, we'll need to change the package to remove the resolutions for the enhanced-resolve dependency.

## Testing instructions

Run

`yarn storybook`

Check that storybook starts on [localhost:6006](http://localhost:6006/)

## Screenshots (optional)

## Reviewers
